SampleGallery/MainPage.xaml.cs

```diff
@@ -323,7 +323,8 @@ public enum SDKVERSION
             _15063,       // Creators Update (1703)
             _16299,       // Fall Creators Update
             _17134,       // Version 1803
-            _17763        // Version 1810
+            _17763,       // Version 1810
+            _18362        // Version 1903
```

Comment:
Put a comma here:  _18362,
That way it won't show up as a change when we add the next version.

While you're here, I think 17763 is really Version 1809, not 1810